.github/workflows/release.yml

```diff
@@ -10,14 +10,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      # Get GitHub token via the CT Changesets App
+      - name: Generate GitHub token (via CT Changesets App)
+        id: generate_github_token
+        uses: tibdex/github-app-token@v2.1.0
+        with:
+          app_id: ${{ secrets.CT_CHANGESETS_APP_ID }}
+          private_key: ${{ secrets.CT_CHANGESETS_APP_PEM }}
```

Comment:
Are these secrets now configured at Github repository level?

Reply:
They are configured on the organisation level actually like other apps. With the addition that they're also restricted to which (six in total) repositories they are exposed. A quite nice feature in GitHub :)